include/nil/crypto3/hash/detail/blake2b/blake2b_policy.hpp

```diff
@@ -31,12 +32,12 @@ namespace nil {
                     constexpr static const std::size_t block_words = block_bits / word_bits;
                     typedef std::array<word_type, block_words> block_type;
 
-                    constexpr static const std::size_t state_bits = 1024;
-                    constexpr static const std::size_t state_words = 2 * state_bits / word_bits;
+                    constexpr static const std::size_t state_bits = 512;
```

Comment:
Blake2b RFC says that there are 8 state words, so I changed state_bits to the proper value.  Here, since word_bits = 64 we have state_words = 32, which is wrong.